tests/html5skin-test.js

```diff
@@ -307,6 +307,31 @@ describe('Controller', function() {
       expect(stopBufferingTimerSpy.callCount).toBe(8);
       expect(controller.state.bufferingTimer).toBeFalsy();
     });
+
+    it('should ads played, device was moved - video should continue to play in last orientation', function() {
```

Comment:
Can we move this to a more appropriate section? This seems to be on the `Buffering state` tests, which are unrelated